tests/snappi_tests/utils/reporter_factory.py

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
nit: remove empty line.

I wonder why pre-commit didn't fail for this.... CI does failed due to static analysis. might be better to check that.